src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,19 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment: { name, email, body } }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">
+        {name}
+      </strong>
+      {' by '}
```

Comment:
it's better just to wrap with a `span`
```suggestion
      <span> by </span>
```